README.md

```diff
@@ -2,10 +2,9 @@
 
 [![Build Status](https://travis-ci.org/fluo-io/webindex.svg?branch=master)](https://travis-ci.org/fluo-io/webindex)
 
-An example Fluo applications that indexes links to web pages in multiple ways.
-The example uses CommonCrawl data as input.  See the
-[tables](docs/tables.md) and [code](docs/code-guide.md) documentation for
-more information about how this example works.
+Webindex is an example Fluo application that indexes links to web pages in multiple ways.
+The example uses [CommonCrawl][cc] data as input.  See the [tables](docs/tables.md) and 
+[code](docs/code-guide.md) documentation formore information about how this example works.
```

Comment:
it might worthwhile to mention that this example is the most complicated one.... and not a good place to start.